account_commission/models/account_move.py

```diff
@@ -56,7 +56,7 @@ def button_draft(self):
             raise exceptions.ValidationError(
                 _("You can't cancel an invoice with settled lines"),
             )
-        self.mapped("line_ids.settlement_id").write({"state": "except_invoice"})
+        self.mapped("line_ids.settlement_id").write({"state": "invoiced"})
```

Comment:
@AungKoKoLin1997 Please read the review comment again - we should take care of button_cancel() and unlink() as well.